src/garage/np/algos/off_policy_rl_algorithm.py

```diff
@@ -65,7 +65,7 @@ def __init__(self,
         self.min_buffer_size = min_buffer_size
         self.rollout_batch_size = rollout_batch_size
         self.reward_scale = reward_scale
-        self.evaluate = False
+        self._first_buffer_size_steps_done = False
```

Comment:
This tracks whether the replay buffer is at least `min_buffer_size` right? So why not just test that directly?

`self.replay_buffer.n_transitions_stored >= self.min_buffer_size`

Perhaps the original author was concerned about having such a long conditional everywhere? If so, you can add a helper property:

```python
@property
def _buffer_prefilled(self):
    return self.replay_buffer.n_transitions_stored >= self.min_buffer_size
```